make/jdk/src/classes/build/tools/classlist/HelloClasslist.java

```diff
@@ -160,13 +166,22 @@ record B(int b) { }
             case B b -> b.b;
             default -> 17;
         };
+        // record run-time methods
+        o.equals(new B(5));
+        o.hashCode();
         LOGGER.log(Level.FINE, "Value: " + value);
 
         // The Striped64$Cell is loaded rarely only when there's a contention among
         // multiple threads performing LongAdder.increment(). This results in
         // an inconsistency in the classlist between builds (see JDK-8295951).
         // To avoid the problem, load the class explicitly.
         Class<?> striped64Class = Class.forName("java.util.concurrent.atomic.Striped64$Cell");
+
+        // Initialize FFM linkers
+        var signature = FunctionDescriptor.of(ADDRESS,
+                ADDRESS, ADDRESS, ADDRESS,
+                JAVA_BOOLEAN, ADDRESS, ADDRESS);
+        Linker.nativeLinker().downcallHandle(signature);
```

Comment:
Why do we care specifically about this shape, but not about others?

Reply:
That was from the past attempt of linking Class forName0. Tge exact shape doesn't really matter here as we are mainly saving on DMH.

Reply:
Can I just change this signature to `FunctionDescriptor.ofVoid()`?

Reply:
Ok, that makes sense. I suppose in the future we might want to pre-generate BindingSpecializer classes, and then it would be useful to figure out which shapes are most common.

I think using `ofVoid()` for now sounds good.